azkaban-common/src/test/java/azkaban/executor/MockExecutorLoader.java

```diff
@@ -54,7 +54,8 @@
   @Override
   public void uploadExecutableFlow(ExecutableFlow flow)
       throws ExecutorManagerException {
-    flows.put(flow.getExecutionId(), flow);
+    ExecutableFlow exFlow = ExecutableFlow.createExecutableFlowFromObject(flow.toObject());
```

Comment:
> would a Mockto based Mock ExecutorLoader which does nothing work and with less complications?

It would be much better. I'll try to switch to that..